package.yaml

```diff
@@ -83,6 +83,7 @@ dependencies:
 - fsnotify >= 0.4.1
 - generic-deriving
 - ghc-boot
+- hashable
```

Comment:
You may not need a direct dependency on `hashable` because package `rio` depends on it.

Reply:
I need to import
```
import           Data.Hashable ( Hashable(..) )
```
So I don't think so

Reply:
`Stack.Prelude` re-exports `RIO` and `RIO` exports `Hashable`?

Reply:
My bad, I thought it didn't reexport the methods

Reply:
You may well be correct; I'll have a look.

Reply:
I think you are correct.

Reply:
It seems I was :

> Module ‘RIO’ does not export ‘Hashable(hashWithSalt)’typecheck

While simply importing Hashable works (I find it odd to only reexport the class by the way, but...)

Reply:
The RIO project's reasoning is here: https://github.com/commercialhaskell/rio/pull/219.